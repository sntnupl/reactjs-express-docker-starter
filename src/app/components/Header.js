import React from 'react';
import { NavLink } from 'react-router-dom';

const Header = () => (
    <div className="app-hdr">
        <ul>
            <li>
                <NavLink
                    to="/"
                    activeClassName="is-active"
                    exact={true}>Home</NavLink>
            </li>
            <li>
                <NavLink
                    to="/forecast"
                    activeClassName="is-active"
                    exact={true}>Forecast</NavLink>
            </li>
            <li>
                <NavLink
                    to="/about"
                    activeClassName="is-active"
                    exact={true}>About</NavLink>
            </li>
        </ul>
    </div>
);

export default Header;
