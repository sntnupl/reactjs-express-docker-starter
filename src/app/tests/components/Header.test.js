import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import Header from '../../components/Header';

test('should render Header correctly', () => {
    const wrapper = shallow(<Header/>);
    expect(toJSON(wrapper)).toMatchSnapshot();
});


