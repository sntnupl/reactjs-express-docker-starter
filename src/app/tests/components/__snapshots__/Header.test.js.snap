// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Header correctly 1`] = `
<div
  className="app-hdr"
>
  <ul>
    <li>
      <NavLink
        activeClassName="is-active"
        ariaCurrent="true"
        exact={true}
        to="/"
      >
        Home
      </NavLink>
    </li>
    <li>
      <NavLink
        activeClassName="is-active"
        ariaCurrent="true"
        exact={true}
        to="/forecast"
      >
        Forecast
      </NavLink>
    </li>
    <li>
      <NavLink
        activeClassName="is-active"
        ariaCurrent="true"
        exact={true}
        to="/about"
      >
        About
      </NavLink>
    </li>
  </ul>
</div>
`;
